backend/api/python_http_client/kfp_server_api/__init__.py

```diff
@@ -21,15 +21,15 @@
 
     This file contains REST API specification for Kubeflow Pipelines. The file is autogenerated from the swagger definition.  # noqa: E501
 
-    The version of the OpenAPI document: 1.0.0-dev.1
+    The version of the OpenAPI document: 1.0.0-rc.1
```

Comment:
Can we prevent changing all those files every time?

Reply:
If we decide to commit these into the repo, they will be automatically updated each release.

Other than that, I don't think that's possible. What concerns do you have?